packages/@aws-cdk/aws-stepfunctions/lib/state-graph.ts

```diff
@@ -55,6 +55,10 @@ export class StateGraph {
      */
     private superGraph?: StateGraph;
 
+    /**
+     *
+     * @param startState state that gets executed when the state machine is launched
+     */
```

Comment:
document both params.

Also avoid the empty first line inside the comment block. Same in other places.

Reply:
done